util/storage-samples/storage-owner-sample/src/main/java/org/fidoalliance/fdo/storage/OwnerSysModule.java

```diff
@@ -146,7 +146,7 @@ private Composite getNextMessage(Composite resource) {
       case FdoSys.KEY_ACTIVE: {
 
         byte[] content = new OwnerDbManager().getSystemResourceContent(dataSource, resId);
-        if (content.length == 1 && content[0] == CBOR_TRUE) {
+        if (content.length == 1 && content[0] == ((byte) CBOR_TRUE)) {
```

Comment:
Instead of explicitly casting CBOR_TRUE to byte, I think it's better to change the CBOR_TRUE declaration from int to byte.